org-formation/090-systems-manager/QuickSetup-PatchPolicy-Prod.yaml

```diff
@@ -0,0 +1,56 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: An SSM patch policy to patch all instances in the given OUs
+Parameters:
+  TargetOrgUnits:
+    Type: String
+    Description: >-
+      Comma separated list of AWS Organizational Units to target.
+  TargetRegions:
+    Type: String
+    Description: >-
+      Comma separated list of AWS Organizational Units to target.
```

Comment:
it's probably better to change these to [CommaDelimitedList](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html#parameters-section-structure-properties)  types

Reply:
I don't think that's appropriate, that would unpack the input into a list of strings, but the parameters these are being passed to expect a comma-delimited string, not a list of strings, so I'm pretty sure I would have to repack the values into a single string with `!Join` if this were a `CommaDelimitedList`